test/functional/http/get/transactions.js

```diff
@@ -865,5 +913,46 @@ describe('GET /api/transactions', () => {
 				});
 			});
 		});
+		/* eslint-disable mocha/no-skipped-tests */
```

Comment:
Can we add a comment clarifying why this exact test is skipped and when will be unskipped?

Reply:
good idea!
I added the comment now